src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -166,6 +167,11 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                         displayHelp = true;
                         continue;
 
+                    case "version":
+                        displayVersion = true;
+                        displayLogo = false;
```

Comment:
Those are the semantics of this switch, by design. Are you saying that you prefer that be enforced behaviorally without modifying the flags in this way? One can toggle `displayLogo` after specifying `displayVersion` though I don't know why you would want to. Do you feel that should be forbidden, or ignored?